client/package.json

```diff
@@ -88,6 +89,16 @@
     ],
     "commands": [
       {
+        "command": "extension.puppetRestartSession",
```

Comment:
These need to also be added to the command palette list. Lots of duplication in package.json